CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ a release.
 ---
 
 ## [Unreleased]
+
+## [3.5.0] - 2022-01-10
 ### Added
 - SoftDeleteable: Support to use annotations as attributes on PHP >= 8.0.
```

Comment:
Please, fix the indentation.

Reply:
hmm how?

Reply:
Forget about it. I can confirm the indentation is fine by checking out the commit locally.
Maybe there is a visual issue with the Github UI:
![image](https://user-images.githubusercontent.com/1231441/148845613-e10d54d4-ef1a-414d-a020-c50cfc794ef6.png)

Reply:
🤔 looks like it's when there is no space at the beginning, it adds like a padding to be able to show the ➕ icon on hover.